src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -63,7 +71,7 @@ class PipelineRunSchema(NamedSchema, table=True):
     )
 
     # Foreign keys
-    deployment_id: UUID = build_foreign_key_field(
+    deployment_id: Optional[UUID] = build_foreign_key_field(
```

Comment:
Why is this optional now? The mock run creation still provides a deployment and your alembic migration doesn't make this optional either

Reply:
Actually it always was, just with a wrong type annotation. If you take a look at the `build_foreign_key_field` function, we pass `nullable=True` there. This is even necessary, as pipeline runs without deployments exist from a time before we introduced these deployments.